lib/linearizer.js

```diff
@@ -230,7 +230,7 @@ module.exports = class Linearizer {
   }
 
   // ack if any head is closer to confirming a value
-  _shouldAckHeads (writer) {
+  _shouldAckHeads (writer, pending) {
```

Comment:
pending is passed down somewhere already?

Reply:
yeh it's already being passed down here: https://github.com/holepunchto/autobase/blob/main/lib/linearizer.js#L209